resources/manifest_schema.json

```
@@ -0,0 +1,156 @@
+{
+  "anyOf": [
+    {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "$ref": "#/project"
+      }
+    },
+    {
+      "$ref": "#/project"
+    }
+  ],
```

Nice that this is possible :+1:

---

took me some time to find but i think it's cool to 
there is a little issue when tracking errors with this one:
when ever u find an error while parsing it throws two errors, one is that is doesn't match the first schema provided and the other error is the actual error
kind of confusing but im not sure if we should keep it that way or try to hide the error for "not an array" if its not an array anyways

---

`"validation_error": {
"sub_errors": [
  "{'titdle': 'test_project0', 'url': ['www.github.com/knex/'], 'analysis': [{'url': 'here-you-find-the-analysis.com', 'description': 'thats what this analysis is about'}], 'authors': [{'name': 'test_owner0', 'email': 'email@address.com'}], 'team': {'team_name': 'teamname'}, 'status': 'DONE', 'description': 'can this be a multiline description?', 'date_creation': '1970-01-01', 'date_last_updated': '2017-05-11', 'tags': ['tag', 'bio'], 'future_work': 'this is something related to this work', 'related_projects': [{'id': 0, 'name': 'name of the listed project', 'author': 'author', 'url': 'www.my.url'}]} is not of type 'array'",
  "Additional properties are not allowed ('titdle' was unexpected)",
  "'title' is a required property"
],

`
`
"upper_errors": [
  "{'titdle': 'test_project0', 'url': ['www.github.com/knex/'], 'analysis': [{'url': 'here-you-find-the-analysis.com', 'description': 'thats what this analysis is about'}], 'authors': [{'name': 'test_owner0', 'email': 'email@address.com'}], 'team': {'team_name': 'teamname'}, 'status': 'DONE', 'description': 'can this be a multiline description?', 'date_creation': '1970-01-01', 'date_last_updated': '2017-05-11', 'tags': ['tag', 'bio'], 'future_work': 'this is something related to this work', 'related_projects': [{'id': 0, 'name': 'name of the listed project', 'author': 'author', 'url': 'www.my.url'}]} is not valid under any of the given schemas"
],
}`

---

but only title is written wrong but it takes some to figure it out when reading the msg

---

i guess we should keep both, to make it obvious both are allowed